tools/autodeployment/builds.js

```diff
@@ -73,19 +73,24 @@ function run() {
   // Promote the next build to current and start it
   const build = builds.pending;
   const { cache } = build;
+  const { GITHUB_TOKEN } = process.env;
 
   builds.current = build;
   builds.pending = null;
 
   logger.debug('run() called, starting pending build');
-  build.proc = shell.exec(`./deploy.sh ${build.type} ${build.sha}`, { silent: true }, (code) => {
-    build.finish(code);
-    builds.previous = builds.current;
-    builds.current = null;
-
-    // See if there's another build ready to go
-    run();
-  });
+  build.proc = shell.exec(
+    `./deploy.sh ${build.type} ${build.sha} ${GITHUB_TOKEN}`,
```

Comment:
@manekenpix is this how I would pass the variable into the script?

Reply:
Yes, but you also need the changes inside of the script.

Reply:
Like this? 
https://github.com/Seneca-CDOT/telescope/pull/2863/files#diff-f9bb2ef9263f3b3ba073e76bf32cbf8afb5c4dc70e65156af4078ddc6963ec7eR37